boards/DIM/Src/App/App_ErrorTable.c

```diff
@@ -0,0 +1,216 @@
+#include <stdlib.h>
```

Comment:
`{App|Io}_ErrorTable.{c|h}` are meant to be auto-generated and placed inside `cmake-build-{embedded|gtest}`. I hand-wrote it for now as a prototype.

Reply:
Why are there per-board error tables but also the shared one?

Reply:
The intention was to have only one error table shared across the whole car. I think it became confusing when i decided to put the code in `DIM` rather than in `shared`. Just to be clear, I _don't_ want a per-baord error table anywhere.